### PR TITLE
Add docs and better warnings for pdflatex depend

### LINKED ISSUE #466: LaTeX not listed as a dependency for building docs in installation instructions
# Latex not listed as a dependency for building docs in installation instructions

### Background
- LaTeX is used to build pdf docs
- When building docs without LaTeX the process fails
- LaTeX is not listed as a dependency for building docs in any installation instructions

### Alternative Solutions
* Do nothing and leave users to install after encountering the error
* Warn and pass instead of failing critically 
  
### Environment
*  Tested using the Dockerfile from #452, run `./docs/build_docs.sh` after `pip install -e "$GEOIPS_PACKAGES_DIR/geoips[doc]"`

Output: 

```
ERROR: latex must be installed in order to create pdf documentation.
  try 'module load latex'
```

### PR DESCRIPTION
**Pull request by biosafetylvl5**
* [x] NO REQUIRED ***existing tests*** (explain why not required)
* [x] NO REQUIRED ***unit tests*** (explain why not required)
* [x] NO REQUIRED ***integration tests*** (explain why not required)
* [x] Required ***documentation*** added for new/modified functionality
* [x] Required ***release notes***  will be added
* [x] NO REQUIRED ***updates to other repos*** (explain why not required)

# Related Issues
Fixed #466 

# Summary
* Added clarification regarding pdflatex dependency for building PDF documentation.

# Testing Instructions
No exhaustive testing is required. This is a documentation/readability fix.

### CODE REVIEW

File: docs/build_docs.sh [hunk 1]
```diff
@@ -193,7 +193,9 @@ if [[ "$pdf_required" == "True" ]]; then
     which_latex=`which latex`
     if [[ "$which_latex" == "" ]]; then
         echo "ERROR: latex must be installed in order to create pdf documentation."
-        echo "  try 'module load latex'"
+        echo "  try 'conda install latexcodec' if in anaconda"
```
Comment (mindyls) [inline docs/build_docs.sh:196]: @biosafetylvl5  Note I updated this from module load latex to conda install latexcode - that is the actual conda command required to install latex.

**Review by mindyls — APPROVED** [review]
Looks good to me - just note I updated module load latex to conda install latexcodec for the log statement if latex is not found.

File: docs/build_docs.sh (hunk 1 above)
Comment (biosafetylvl5) [reply docs/build_docs.sh:196]: Thanks! What was `module load latex` from?

Comment (jsolbrig) [reply docs/build_docs.sh:196]: `module load` is a command used on some linux systems to "load" a plugin by changing environment variables like Paths.

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**